dumb.coffee.js

```diff
@@ -0,0 +1,3 @@
+class this_isClearly_Wrong
+    hello_There = ->
```

Comment:
Identifier 'hello_There' is not in camel case.<br>Expected '{' and instead saw 'hello_There'.<br>Unexpected '='.<br>Unexpected '-'.<br>Unexpected '>'.